pkg/tmpl/context_funcs_test.go

```diff
@@ -307,14 +307,44 @@ func TestSetValueAtPath_TwoComponents(t *testing.T) {
 }
 
 func TestTpl(t *testing.T) {
-	text := `foo: {{ .foo }}
-`
-	expected := `foo: FOO
-`
 	ctx := &Context{basePath: "."}
-	actual, err := ctx.Tpl(text, map[string]interface{}{"foo": "FOO"})
-	require.NoError(t, err)
-	require.Equal(t, expected, actual)
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+		hasErr   bool
+		data     map[string]interface{}
+	}{
+		{
+			name:     "simple",
+			input:    `foo: {{ .foo }}`,
+			expected: `foo: Foo`,
+			data: map[string]interface{}{
+				"foo": "Foo",
+			},
+		},
+		{
+			name: "multiline",
```

Comment:
```suggestion
			name: "multiline_input",
```

We could differentiate a multiline input vs a multiline value in data this way. Just my two cents!